<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>DarknessForest</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="menuWidget">
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/DarkForest.jpg);</string>
    </property>
    <widget class="QPushButton" name="newGemeButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>130</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>50</height>
      </size>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
font: 75 12pt &quot;Yu Gothic&quot;;
</string>
     </property>
     <property name="text">
      <string>New Game</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loadGameButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>180</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
font: 75 12pt &quot;Yu Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>Load Game</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveGameButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>230</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
font: 75 12pt &quot;Yu Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>Save Game</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exitGameButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>280</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
font: 75 12pt &quot;Yu Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>340</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
color: white;
font: italic 36pt &quot;Monotype Corsiva&quot;;
</string>
     </property>
     <property name="text">
      <string>DarknessForest</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="newGameWidget"/>
   <widget class="QWidget" name="loadGameWidget"/>
   <widget class="QWidget" name="saveGameWidget"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
